.github/workflows/main.yml

```diff
@@ -121,6 +121,12 @@ jobs:
         run: |
           npm i
           npm run get-proposals
+      - name: "⚙️ npm — docsy"
+        # Install dependencies according to https://github.com/google/docsy/#prerequisites
+        run: |
+          cd themes/docsy
+          npm i
+          npm i --save-dev autoprefixer postcss-cli postcss
```

Comment:
>           npm i --save-dev autoprefixer postcss-cli postcss

this doesn't look right at all. This is modifying the `package.json` of docsy to say that these libraries are dev-time dependencies of docsy. If that's true, then it should be checked into git rather than done on the fly... but I don't think it's true.

The suggestion at https://github.com/google/docsy/#prerequisites is to do it in the Hugo site which uses docsy (ie, `matrix-spec`), not docsy itself. And it's already been done in matrix-spec: https://github.com/matrix-org/matrix-spec/blob/main/package.json#L21-L26

Reply:
Done in 3313b6a0a5a8fafae0479b8513859e0201d62ff0.